rackcity/models/network_port.py

```diff
@@ -84,7 +97,22 @@ class Meta:
             ),
         ]
 
-    def save(self, *args, **kwargs):
-        if self.mac_address is not None:
-            self.mac_address = format_mac_address(self.mac_address)
-        super(NetworkPort, self).save(*args, **kwargs)
+
+class NetworkPortCP(AbstractNetworkPort):
+    asset = models.ForeignKey(
+        AssetCP,
+        on_delete=models.CASCADE,
+        verbose_name="asset",
+    )
+    change_plan = models.ForeignKey(
+        ChangePlan,
+        on_delete=models.CASCADE,
+    )
```

Comment:
is it redundant to store change plan here? shouldn't it always be the same change plan as `asset.change_plan`?

Reply:
uniqueness constraint? but not sure